packages/adapter-dynamodb/package.json

```
@@ -1,19 +1,30 @@
 {
   "name": "@next-auth/dynamodb-adapter",
   "repository": "https://github.com/nextauthjs/next-auth",
-  "version": "1.2.0",
+  "version": "2.0.0-experimental.0",
```

```suggestion
  "version": "2.0.0",
```

When this gets merged, `monorepo-release` takes over versioning, and since it does not have a concept of `experimental`, we could just leave it as the old version.

Since `2.0.0` was [released](https://www.npmjs.com/package/@next-auth/dynamodb-adapter/v/2.0.0) by accident already though, we can label this as `fix` (in the PR title) to be released as a patch (`2.0.1`).

Then we can manually change the release notes after, to note that this was a breaking change release.